src/draw/lv_draw_buf.c

```diff
@@ -154,30 +154,32 @@ void lv_draw_buf_flush_cache_user(const lv_draw_buf_handlers_t * handlers, const
 void lv_draw_buf_clear(lv_draw_buf_t * draw_buf, const lv_area_t * a)
 {
     LV_ASSERT_NULL(draw_buf);
-    if(a && lv_area_get_width(a) < 0) return;
-    if(a && lv_area_get_height(a) < 0) return;
 
     const lv_image_header_t * header = &draw_buf->header;
     uint32_t stride = header->stride;
 
     if(a == NULL) {
-        /*Need skip the palette if exists*/
-        uint8_t * bufc = lv_draw_buf_goto_xy(draw_buf, 0, 0);
-        lv_memzero(bufc, header->h * stride);
-        return;
```

Comment:
It is recommended to keep the `return`, so that there are fewer `else` and the process is clearer.